src/common/types/util.types.ts

```diff
@@ -46,22 +46,16 @@ export enum ParachainStatus {
   Shutdown
 }
 
+export type Price = {
+  usd: number;
+};
+
 export type Prices = {
-  bitcoin:
-    | {
-        usd: number;
-      }
-    | undefined;
-  relayChainNativeToken:
-    | {
-        usd: number;
-      }
-    | undefined;
-  governanceToken:
-    | {
-        usd: number;
-      }
-    | undefined;
+  bitcoin: Price | undefined;
+  relayChainNativeToken: Price | undefined;
+  governanceToken: Price | undefined;
+  wrappedToken: Price | undefined;
+  [token: string]: Price | undefined;
```

Comment:
note: this is a short term fix to avoid refactoring prices across the application. There's a task in the next sprint to move the price api call to a hook, and refactor the `getCollateralPrice` utility to return any price. Part of that will involve refactoring this type and introducing greater type safety.